.github/workflows/all_plugins.yaml

```diff
@@ -26,20 +26,23 @@ jobs:
         with:
           channel: 'stable'
 
-      - name: "Setup and install dependencies"
-        run: ./scripts/update-packages
+      - name: "Install Tools"
+        run: |
+          ./.github/workflows/scripts/install-tools.sh
+      
+      - name: "Bootstrap Workspace"
+        run: melos bootstrap
+        
+      - name: "Get dependencies"
+        run: melos run get
 
-      - name: "Run analyze on stripe"
-        run: cd stripe && flutter analyze && cd ..
-
-      - name: "Run analyze on stripe_android"
-        run: cd stripe_android && flutter analyze && cd ..
-
-      - name: "Run analyze on stripe_ios"
-        run: cd stripe_ios && flutter analyze && cd ..
-
-      - name: "Run analyze on stripe_platform_interface"
-        run: cd stripe_platform_interface && flutter analyze && cd ..
+      - name: "Run Analyze"
+        run: melos run analyze
+     
+      - name: "Pub Check"
+        run: |
+          melos exec -c 1 --no-private --ignore="*example*"  -- \
+            pub publish --dry-run
```

Comment:
While you are on this, would you mind reviewing the TODO that I created at line 17-18?
https://github.com/flutter-stripe/flutter_stripe/pull/56/files#diff-61a085d0578afbe6a13da0c8926054ea59e83ce30bd655a6349e78afe7940351R17

Thanks!!